docs/k8s-cni-kube-ovn.md

```diff
@@ -6,7 +6,7 @@ and easy-to-use network solution for Kubernetes.
 
 ## Prerequisites
 
-The override values file for Kube-OVN can be found in `/etc/genestack/helm-configs/kube-ovn/kube-ovn-helm-overrides.yaml`
+The override values file for Kube-OVN can be found in `/etc/genestack/base-helm-configs/kube-ovn/kube-ovn-helm-overrides.yaml`
```

Comment:
If we're trying to point at the original values file, we can get there via `/opt`.

```suggestion
The override values file for Kube-OVN can be found in `/opt/genestack/base-helm-configs/kube-ovn/kube-ovn-helm-overrides.yaml`
```